src/firebolt/client/client.py

```diff
@@ -20,6 +21,15 @@
     mixin_for,
 )
 
+# Explicitly import all available backend not get into
+# anyio race condition during backend import
+for backend in ("asyncio", "trio"):
+    try:
+        get_asynclib(backend)
+    except ModuleNotFoundError:
+        # Not all backends might be installed
+        pass
```

Comment:
I'm just worried since we're using a "private" API for this hack it might change without notice. As we're pinning the httpx version we're fine for now, but if we were to upgrade it might cause issues. Hopefully it will be picked up by nightly.

Reply:
I will investigate how this could be properly fixed